### PR TITLE
Release 2.12.1

### PR DESCRIPTION
**Pull request by submarine-launched**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-client",
-  "version": "2.11.0",
+  "version": "2.12.2",
```
Comment (submarine-launched) [inline package.json:3]: Not sure if we want this to be 2.12.2, the version is should be 2.12.1

File: package.json [hunk 2]
```diff
@@ -13,8 +13,6 @@
   "scripts": {
     "build": "gulp build",
     "start": "http-server ./demo",
-    "gulpversion": "gulp build && git add .",
-    "prepublish": "typings install && gulp build",
```
Comment (submarine-launched) [inline package.json:17]: This makes it impossible to install the package locally from package.json without typings globally installed.

Comment (ali-hamud) [reply package.json:17]: Why? It's just omitting prepublish step. users do not publish packages so they do not run this step.

File: package.json (hunk 1 above)
Comment (ali-hamud) [reply package.json:3]: We are releasing 2.12.2.

**Comment by ali-hamud** [comment]
Let's wait with this.

**Pull request closed without merging**